papis/git.py

```diff
@@ -33,9 +35,8 @@ def commit(path: str, message: str) -> None:
     :param path: Folder where a git repo exists.
     :param message: Commit message
     """
-
     logger.info("Committing '%s' with message '%s'.", path, message)
-    cmd = "git commit -m '{}'".format(message)
+    cmd = "git commit -m {}".format(shlex.quote(message))
```

Comment:
This looks good to me! 

`add` and `remove` also seem to do some quoting of their own. If you have the time, can you check if those need any special handling too?

Reply:
I never had Issues with adding or removing, but since escaping the string shouldn't break it and make it more robust, I added it to `add` and `remove`. I tested it quickly and it works without issues 👍🏽